third_party/2and3/google/protobuf/internal/containers.pyi

```diff
@@ -19,14 +19,33 @@ class BaseContainer(Sequence[_T]):
     @overload
     def __getitem__(self, key: slice) -> List[_T]: ...
 
-class RepeatedScalarFieldContainer(Generic[_T], BaseContainer[_T]):
+class RepeatedScalarFieldContainer(BaseContainer[_T]):
     def __init__(self, message_listener: MessageListener, message_descriptor: Descriptor) -> None: ...
+    def append(self, value: _T) -> None: ...
+    def insert(self, key: int, value: _T) -> None: ...
+    def extend(self, elem_seq: Optional[Iterable[_T]]) -> None: ...
     def MergeFrom(self, other: RepeatedScalarFieldContainer[_T]) -> None: ...
+    def remove(self, elem: _T) -> None: ...
+    def pop(self, key: int = ...) -> _T: ...
+    @overload
+    def __setitem__(self, key: int, value: _T) -> None: ...
+    @overload
+    def __setitem__(self, key: slice, value: Iterable[_T]) -> None: ...
+    def __getslice__(self, start: int, stop: int) -> List[_T]: ...
+    def __setslice__(self, start: int, stop: int, values: Iterable[_T]) -> None: ...
+    def __delitem__(self, key: Union[int, slice]) -> None: ...
+    def __delslice(self, start: int, stop: int) -> None: ...
```

Comment:
I think this is a typo, it should be `__delslice__` (trailing underscores missing).